sr/sr.go

```diff
@@ -267,7 +267,12 @@ func (sr *SR) writeResults(outfile string, layers []int, requestChan chan result
 		}
 
 		// Add InMAP data
-		data, err := sr.d.Results(true, false, inmapVars)
+		o, err := inmap.NewOutputter("", true, inmapVars, nil)
+		if err != nil {
+			errChan <- fmt.Errorf("preparing output variables: %v", err)
```

Comment:
Ah I see. This is to make it clear that the error is occurring outside of the SR code.